4-Services/HeroApp/HeroData/Repositories/SuperHeroRepository.cs

```diff
@@ -18,33 +18,39 @@ public SuperHeroRepository(Entities.SuperHeroContext context)
         }
         public void AddSuperHero(SuperHero superHero)
         {
-            throw new NotImplementedException();
+            context.SuperHeroes.Add(mapper.Map(superHero));
+            context.SaveChanges();
         }
 
         public void DeleteSuperHero(int id)
         {
-            throw new NotImplementedException();
+            var superHero = context.SuperHeroes.FirstOrDefault(x => x.Id == id);
+            context.SuperHeroes.Remove(superHero);
+            context.SaveChanges();
         }
 
         public SuperHero GetSuperHeroById()
```

Comment:
This method should be parameterised to accept different ids instead of searching for hardcoded values.